cmd/createtree/main_test.go

```diff
@@ -136,6 +151,22 @@ func TestRun(t *testing.T) {
 			wantErr: true,
 		},
 		{
+			desc:    "PEMKeyFile",
+			opts:    &pemKeyOpts,
+			wantErr: false,
+			wantTree: &trillian.Tree{
```

Comment:
nit: Making this tree at the start of the test might be nicer, in the same way that nonDefaultTree was created. Then you could get rid of `marshalAny()` as well.